### PR TITLE
Update the CRD schema version

### PR DESCRIPTION
**Pull request by tpapagian**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release_template.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ assignees: ''
 
       export RELEASE=v0.8.1
 
+- [ ] Only for major release, update [CustomResourceDefinitionSchemaVersion](https://github.com/cilium/tetragon/blob/6f2809c51b3fbd35b1de0a178f1e3d0b18c52bcc/pkg/k8s/apis/cilium.io/v1alpha1/register.go#L18) to `vX.Y+1.0`.
+
```
Comment (kkourt) [inline .github/ISSUE_TEMPLATE/release_template.md:18]:
```suggestion
- [ ] If you branch out vX.Y, create a new PR in the `main` branch to update [CustomResourceDefinitionSchemaVersion](https://github.com/cilium/tetragon/blob/6f2809c51b3fbd35b1de0a178f1e3d0b18c52bcc/pkg/k8s/apis/cilium.io/v1alpha1/register.go#L18) to `vX.Y+1.0`.

```

**Review by kkourt — CHANGES_REQUESTED** [review]
Thanks!

File: .github/ISSUE_TEMPLATE/release_template.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tpapagian) [reply .github/ISSUE_TEMPLATE/release_template.md:18]: Why do we need `"Once PR is merged, tag the first commit in master which is not in the `X.Y` branch as `vX.Y+1.0-pre.0`."`?

[the remark this answers is not in the document]
Comment (kkourt) [reply .github/ISSUE_TEMPLATE/release_template.md]: Not sure I'm following. Here's what I'm thinking:

```mermaid

gitGraph
    commit
    commit
    branch "v0.10"
    checkout main
    commit id: "CRD -> v0.11.0" tag: "v0.11.0-pre.0"
    commit
    commit
    checkout "v0.10"
    commit
    commit
    commit tag: "v0.10.1"
```

[the remark this answers is not in the document]
Comment (kkourt) [reply .github/ISSUE_TEMPLATE/release_template.md]: So there is the CRD change commit that we can tag. Makes sense?

**Review by kkourt — APPROVED** [review]

**Pull request merged**